WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -42,6 +42,9 @@ class WPMainActivityViewModel @Inject constructor(
     private val _createAction = SingleLiveEvent<ActionType>()
     val createAction: LiveData<ActionType> = _createAction
 
+    private val _showQuickStarInBottomSheet = MutableLiveData<Boolean>()
+    val showQuickStarInBottomSheet: LiveData<Boolean> = _showQuickStarInBottomSheet
+
```

Comment:
Instead of exposing this live data, you could merge both this and the main actions into one view state model for the whole bottom sheet. It could look like this: 
```
private val _showQuickStarInBottomSheet = MutableLiveData<Boolean>()

    private val _mainActions = MutableLiveData<List<MainActionListItem>>()
    val mainActions: LiveData<List<MainActionListItem>> = merge(_mainActions, _showQuickStarInBottomSheet) { mainActions, showQuickStart ->
        if (showQuickStart != null && mainActions != null) {
            mainActions.map { if (it is CreateAction && it.actionType == CREATE_NEW_POST) it.copy(showQuickStart = showQuickStart) else it }
        } else {
            mainActions
        }
    }
```
There are a few advantages of this principle. One is that you can see the amount of LiveData already existing in this class. This helps us keep everything related in one place. It's easy to check the state of the whole component if everything is in one place. It's also very easy to test the whole state. You can test both that an action is present and that it's showing the quick start. 

This example requires something like this in the `CreateAction`
```
data class CreateAction(
        override val actionType: ActionType,
        @DrawableRes val iconRes: Int,
        @StringRes val labelRes: Int,
        val onClickAction: ((actionType: ActionType) -> Unit)?,
        val showQuickStart: Boolean = false
    ) : MainActionListItem()
```